style.css

```diff
@@ -0,0 +1,838 @@
+h1 {
```

Comment:
Try to avoid` position` in all cases. Use it, if it is really necessary

Reply:
If you don't know flex, you can use `text-align: center` or `margin: 0 auto` for centralizing elements or text